packages/actions/src/Imports/Http/Controllers/DownloadImportFailureCsv.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Filament\Actions\Imports\Http\Controllers;
+
+use Filament\Actions\Imports\Models\FailedImportRow;
+use Filament\Actions\Imports\Models\Import;
+use League\Csv\Writer;
+use SplTempFileObject;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+class DownloadImportFailureCsv
+{
+    public function __invoke(Import $import): StreamedResponse
+    {
+        abort_unless($import->user->is(auth()->user()), 403);
```

Comment:
Might make sense to add more fine-grained control?
Not sure if that's the case, but can any user access the download route, since it's not scoped to a single panel?

Reply:
1) This URL is only exposed when the "import completed" notification happens
2) The "import completed" notification only gets sent to `$import->user`

You can bind a new controller to the container if you want to replace this

Reply:
Still a security issue though, even though the route is only exposed temporarily.
Can't we just check `$import->user` here?

Reply:
Why not use a signed temporary URL for this?

Reply:
What's the security issue?

We should not use a temporary URL as we cannot guarantee when the user is going to download the CSV, and the URL needs to be stored in the database notifications database.

Reply:
Sorry, I was just being blind, lol.
No issues here.